lorien/Misc/I18nParser.gd

```diff
@@ -4,8 +4,18 @@ class_name I18nParser
 const I18N_FOLDER := "res://Assets/I18n/"
 const StringTemplating := preload("res://Misc/StringTemplating.gd")
 
+# -------------------------------------------------------------------------------------------------
+var _first_load := true
```

Comment:
Locales are reloaded multiple times now. This makes it so that the "Laded i18n file"-messages are only logged the first time.